### ISSUE TITLE
Make a Dockerfile?

### ISSUE THREAD

**Issue by carmolim** [Gigamick/burnernote#7]
I was looking for something exactly like that, but I can be 100% that the open-source code is running on your server. To remediate that would be nice to offer a Dockerfile that way everyone could self-host the service.

**Comment by Gigamick** [comment]
Yeah this is a problem with all open source software I guess. You're more than welcome to add some kind of pull request with.... I dunno how you would do it? Some kind of hash or encrypted value or something and you could see it deployed to production?

Other than that, I absolutely assure you that it's the same code in the repo that's in prod lol

**Comment by Gigamick** [comment]
> I was looking for something exactly like that, but I can be 100% that the open-source code is running on your server. To remediate that would be nice to offer a Dockerfile that way everyone could self-host the service.

I am not skilled in docker. If you wanted to PR a docker file I would be happy to add it to merge it.
